alpaca_trade_api/stream.py

```diff
@@ -1,8 +1,3 @@
-"""
-Stream V2
```

Comment:
Should we keep `Stream V2` here? I'm fine with remove the rest of the comments, but I wonder if it's better to state this is V2 explicitly.

Reply:
Actually it isn't. Crypto is in v1beta1, so is the news stream.